src/abstract-classes/browser-plugin.ts

```diff
@@ -140,15 +140,38 @@ export abstract class BrowserPlugin<
     async launch(
         launchContext: LaunchContext<Library, LibraryOptions, LaunchResult, NewPageOptions, NewPageResult> = this.createLaunchContext(),
     ): Promise<LaunchResult> {
-        const { proxyUrl } = launchContext;
+        const { proxyUrl, launchOptions }: {proxyUrl?: string, launchOptions: any} = launchContext;
 
         if (proxyUrl) {
             await this._addProxyToLaunchOptions(launchContext);
         }
+        // This will set the args for chromium based browsers to hide the webdriver.
+        launchOptions.args = this._mergeArgsToHideWebdriver(launchOptions.args);
 
         return this._launch(launchContext);
     }
 
+    /**
+     * @private
+     * @param originalArgs
+     * @returns
+     */
```

Comment:
```suggestion
```

this has no benefit

Reply:
I added because of the `private` to hide it from the docs.